engine: Fix for console  sidebar (#776)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc

= rpk connect install

Install Redpanda Connect.

> This command install the latest version by default. Alternatively, you may specify a Redpanda Connect version using the `--connect-version` flag.

Reviewer: ```suggestion
By default, this command installs the latest version of Redpanda Connect. Alternatively, you can use the command with the `--connect-version` flag to install a specific version.
```

> You may force the installation of Redpanda Connect using the `--force` flag.

Reviewer: ```suggestion
To force the installation of Redpanda Connect, use the `--force` flag.
```

== Usage

[,bash]
----
rpk connect install [flags]
----

== Flags

[cols="1m,1a,2a"]
|===
|*Value* |*Type* |*Description*

> |--connect-version |string |Redpanda Connect version to install. For example: "4.32.0" (default "latest").

Reviewer: ```suggestion
|--connect-version |string | The Redpanda Connect version to install. For example: "4.32.0" (default "latest").
```
